backend/connector-integration/src/connectors/globalpay/transformers.rs

```diff
@@ -285,7 +286,7 @@ pub enum StoredCredentialType {
     Subscription,
 }
 
```

Comment:
Same as above - `Clone` and `Copy` traits added without clear usage.

```suggestion
#[derive(Debug, Serialize)]
```